junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -40,7 +40,12 @@
  * @see Namespace
  */
 @API(status = STABLE, since = "5.0")
-public interface ExtensionContext {
+public interface ExtensionContext extends AutoCloseable {
```

Comment:
I'd like to hide the `close` method, if possible.

Reply:
Sure. Consider it gone.